### ISSUE TITLE
Assigning null string to JSValue doesn't set IsNull

### ISSUE THREAD

**Issue by tarwich** [awesomium/awesomium-pub#19]
If you assign a null value to a variable and assign that variable to a JSValue, then using that JSValue in a JavaScript context is dangerous. It would help if assigning a null value to a JSValue would set the .IsNull or .IsUndefined flag.

``` C#
using System;
using Awesomium.Core;

namespace Simple_Awesomnium_Test
{
    class Program
    {
        static void Main(string[] args)
        {
            String foo;
            JSValue bar;
            JSObject bin = new JSObject();

            foo = "Works";
            bar = foo;
            bin["value"] = bar;

            foo = null; 
            bar = foo;
            bin["value"] = bar; // Doesn't work
            // ^-- An unhandled exception of type 'System.ArgumentNullException' occurred in Awesomium.Core.dll

            if (bar.IsNull) return; // Doesn't help
            if (bar.IsUndefined) return; // Doesn't help either
            if (((String)bar) == null) return; // Fixes the problem
        }
    }
}
```

**Comment by tarwich** [comment]
Does that mean it's fixed?

**Comment by Perikles** [comment]
Yes, this has been fixed since 1.7.5 and 1.7.5.1 (available [here](http://www.awesomium.com/download/)).
